sig-k8s-infra/charter.md

```diff
@@ -0,0 +1,186 @@
+# SIG K8s Infra Charter
+
+This charter adheres to the conventions described in the
+[Kubernetes Charter README] and uses the Roles and Organization Management
+outlined in [sig-governance].
+
+## Scope
+
+The successful migration of ownership and management of all Kubernetes
+project infrastructure from the google.com GCP Organization 
+(or elsewhere) to the CNCF, such that the Kubernetes project is able to
+sustainably operate itself without direct assistance from external vendors
+or entities.
+
+In other words, we seek to eradicate usage of the phrase "oh that's
+something that only an employee of Vendor X can do, we're blocked until
+they respond."
+
+### In scope
+
+#### Code, Binaries and Services
+
+Code, data and policies necessary to provision, update, decommission and
+otherwise manage all project infrastructure. Given that this is a broad scope,
```

Comment:
"project infrastructure" here is quite broad. Is GitHub included in this? Zoom? Slack? Opsgenie?

Reply:
I would say: yes.  But this is more specifically handled by existing SIG Contributor Experience subproject(s), so we delegate

Reply:
An alternative I think we could draw today would be to say: anything that's IaaS is managed by k8s-infra.  Anything else is managed by a more specific SIG, e.g.
- github, slack, discuss: contribex
- netlify: docs
- artifacthub: is a gap I think? release?
- snyk: security
- travis/circleci/codecov/whatever: is up to each subproject owner?

I like this less since it feels more complex and arbitrary, but I am having difficulty thinking of an exception to this as a guiding principle

Reply:
> anything that's IaaS is managed by k8s-infra

I was wondering about this when I read through this originally, especially w/r/t the Areas of responsibility called out below.  I think the bullet points you called out probably makes sense, although `artifacthub`, whatever that ends up being ,  probably should  be  _owned_ by infra, like GCR (or GAR in the future?) is, and then Release would own the artifact promotion, like you called out below? 

If sig-k8s-infra will be the owner of, but will delegate responsibility for, things like Github/Zoom/etc, it would probably be good to explicitly call these out so it is unambiguous?

Reply:
artifacthub = https://artifacthub.io/, ref: https://github.com/kubernetes/steering/issues/188

Reply:
@spiffxp haven't we been adding CNCF folks like ihor as owners in different places?

Reply:
Good point. Are you suggesting I add collaboration with CNCF in the cross cutting section? Something like ensuring CNCF has sufficient credentials to access all the things as a final fallback?

Reply:
Re artifact hub I'm suggesting some part of this project should own the kubernetes org or equivalent there. Not artifact hub itself since it's for all CNCF projects

Reply:
> anything that's IaaS is managed by k8s-infra

That's much more clear and I would be supportive of this. So our GCP/AWS/other cloud provider accounts would roll up into this sig for now.

Reply:
Language that refers to IaaS added here: https://github.com/kubernetes/community/pull/5928/files#diff-b55cad1162db37049c36ff01b2892c020ecd9b8ce4a89284850d9bc2b6a6a25aR28-R34

Language that pushes non-IaaS resources out of scope added here: https://github.com/kubernetes/community/pull/5928/files#diff-b55cad1162db37049c36ff01b2892c020ecd9b8ce4a89284850d9bc2b6a6a25aR184-R187